### PR TITLE
FIX / enabling manifest generation via cli

### PR DESCRIPTION
**Pull request by dgraeber**
### Description:

Enable manifest generation from CLI

### Issue Reference URL

https://github.com/awslabs/aws-orbit-workbench/issues/1247

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: cli/aws_orbit/data/init/default-env-manifest.yaml [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+        InternetAccessible: !ENV ${INTERNET_ACCESSIBLE::true}
+        NodesSubnets: !SSM ${/orbit-f/${name}/resources::NodesSubnets}
+    Frontend:
+        LoadBalancersSubnets: !SSM ${/orbit-f/${name}/resources::LoadBalancersSubnets}
+        SslCertArn: !SSM ${/orbit-f/${name}/resources::SslCertArn}
 Images:
     JupyterUser:
-        Repository: aws-orbit-jupyter-user
-        Source: public
+        Repository: ${account_id}.dkr.ecr.${region}.amazonaws.com/orbit-${name}/jupyter-user
```
Comment (srinivasreddych) [inline cli/aws_orbit/data/init/default-env-manifest.yaml:19]: In few manifests, i have noticed `!ENV ${ACCOUNT_ID::}` and here `${account_id}`. Can we make it consistent?

File: cli/aws_orbit/data/init/default-env-manifest.yaml [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
         Version: latest
-    JupyterUserSpark:
-        Repository: aws-orbit-jupyter-user-spark
-        Source: public
+    OrbitController:
+        Repository: ${account_id}.dkr.ecr.${region}.amazonaws.com/orbit-${name}/orbit-controller
```
Comment (srinivasreddych) [inline cli/aws_orbit/data/init/default-env-manifest.yaml:22]: In few manifests, i have noticed `!ENV ${ACCOUNT_ID::}` and here `${account_id}`. Can we make it consistent?

File: cli/aws_orbit/data/init/default-env-manifest.yaml [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
+        Version: latest
+    UtilityData:
+        Repository: ${account_id}.dkr.ecr.${region}.amazonaws.com/orbit-${name}/utility-data
```
Comment (srinivasreddych) [inline cli/aws_orbit/data/init/default-env-manifest.yaml:25]: In few manifests, i have noticed `!ENV ${ACCOUNT_ID::}` and here `${account_id}`. Can we make it consistent?

Comment (dgraeber) [reply cli/aws_orbit/data/init/default-env-manifest.yaml:25]: No...this is not using the replace function with the env variables....this is specific to this code...there is no !ENV

File: cli/aws_orbit/data/init/default-env-manifest.yaml (hunk 2 above)
Comment (dgraeber) [reply cli/aws_orbit/data/init/default-env-manifest.yaml:22]: No...this is not using the replace function with the env variables....this is specific to this code...there is no !ENV

File: cli/aws_orbit/data/init/default-env-manifest.yaml (hunk 1 above)
Comment (dgraeber) [reply cli/aws_orbit/data/init/default-env-manifest.yaml:19]: No...this is not using the replace function with the env variables....this is specific to this code...there is no !ENV

**Review by srinivasreddych — APPROVED** [review]

**Pull request merged**